### PR TITLE
Adapt admin console for kuzzle v2

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

 - use the new JS SDK 7
 - fix admin console for Kuzzle v2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -93,19 +93,19 @@ jobs:
...
 
 # ---------------------------------------
 # next.console.kuzzle.io
 # ---------------------------------------
     - stage: build & deploy
-      name: "Build & Deploy to next.console.kuzzle.io"
+      name: "Build & Deploy to next-console.kuzzle.io"
 
-      if: type != pull_request && branch =~ ^.[1-9]*-dev$
+      if: type != pull_request && branch =~ ^.[1-9]+-dev$
```
Comment (alexandrebouthinon) [inline .travis.yml]:
```suggestion
      if: type != pull_request && branch =~ ^.[0-9]+-dev$
```

File: src/components/Data/Documents/Page.vue [hunk 2]
```diff
@@ -578,6 +578,7 @@ export default {
 
       let searchQuery = null
       searchQuery = filterManager.toSearchQuery(this.currentFilter)
+      console.log(searchQuery)
```
Comment (Njuelle) [inline src/components/Data/Documents/Page.vue]:
```suggestion
```

File: test/e2e/cypress/integration/users.spec.js [hunk 3]
```diff
@@ -118,7 +118,7 @@ describe('Users', function() {
     cy.contains('User Management')
   })
 
-  it('receives an error when trying to update the mapping for an existing field', function() {
+  it.only('receives an error when trying to update the mapping for an existing field', function() {
```
Comment (alexandrebouthinon) [inline test/e2e/cypress/integration/users.spec.js]:
```suggestion
  it('receives an error when trying to update the mapping for an existing field', function() {
```

**Comment by Aschen** [comment]
Deployed to next-console

**Pull request merged**